SearchAPI/CMR/Output/output_translators.py

```diff
@@ -1,3 +1,4 @@
+from SearchAPI.CMR.Output.asf_search import cmr_to_asf_search, req_fields_asf_search
```

Comment:
Just for consistency, can we reduce this to the relative import? Or if there's a reason to do it this way, we should just update the others to match. 🤷